src/test/java/bio/terra/service/snapshot/SnapshotConnectedTest.java

```diff
@@ -672,4 +798,30 @@ private long queryForCount(
         return countValue.getLongValue();
     }
 
+    private String getFileRefIdFromSnapshot(SnapshotSummaryModel snapshotSummary) throws InterruptedException {
+        Snapshot snapshot = snapshotDao.retrieveSnapshotByName(snapshotSummary.getName());
+        SnapshotDataProject dataProject = dataLocationService.getOrCreateProject(snapshot);
+        BigQueryProject bigQueryProject = BigQueryProject.get(dataProject.getGoogleProjectId());
+
+        StringBuilder builder = new StringBuilder()
+            .append("SELECT file_ref FROM `")
+            .append(dataProject.getGoogleProjectId())
+            .append('.')
+            .append(snapshot.getName())
+            .append(".tableA` AS T") // TODO this is wonky witth "tableA"
+            .append(" WHERE T.file_ref IS NOT NULL LIMIT 1");
+
+        String sql = builder.toString();
+        try {
+            QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(sql).build();
+            TableResult result = bigQueryProject.getBigQuery().query(queryConfig);
+            FieldValueList row = result.iterateAll().iterator().next();
+            FieldValue idValue = row.get(0);
+            String drsUri = idValue.getStringValue();
+            return drsUri;
+        } catch (InterruptedException ie) {
```

Comment:
Do you need both this try/catch for InterruptedException and adding it to the method signature?